docker_logs_cvat.txt

```diff
@@ -0,0 +1,297 @@
+2019-05-23 22:44:15,141 INFO RPC interface 'supervisor' initialized
```

Comment:
Why do we need the file?

Reply:
@nmanovic It's not needed.  I didn't remove it but will.

Reply:
@nmanovic It's gone now.